src/js/data-structures/heap.js

```diff
@@ -43,12 +43,15 @@ export class MinHeap {
     const left = this.getLeftIndex(index);
     const right = this.getRightIndex(index);
     const size = this.size();
-    if (left < size && this.compareFn(this.heap[element], this.heap[left]) > Compare.BIGGER_THAN) {
```

Comment:
@loiane, I got the third edition book but it seems it was still with this issue.